cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -866,7 +866,7 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
             value: {{bicepName .Name}}Identity.outputs.clientId
           }
           {{- end}}
-          {{- if (and .AzureServiceBus .AzureServiceBus.AuthUsingConnectionString) }}
+          {{- if (and (and .AzureServiceBus .AzureServiceBus.AuthUsingConnectionString) (not .AzureServiceBus.IsJms)) }}
```

Comment:
When `.AzureServiceBus.IsJms` equals to true, then `.AzureServiceBus.AuthUsingConnectionString` should be false. This logic should be include in code level instead of template level. I mean revert the change here and add logic in go lang code.